docs/microservices.json

```diff
@@ -516,6 +554,11 @@
                     "id": "document-management-store-api-gateway-web",
                     "hard": true,
                     "apis": []
+                },
+                {
+                    "id": "letters-producer",
```

Comment:
Is CCD also integration with our API.I thought only CMC is integrating with us at the moment?

Reply:
this is cmc

Reply:
formatting made it look like it has been added under ccd :(